app/flame_chart/flame_chart.tsx

```diff
@@ -0,0 +1,532 @@
+import { HorizontalScrollbar } from "buildbuddy/app/components/scrollbar/scrollbar";
```

Comment:
one annoyance with the way we do our internal / external repo crossover - all of our TS paths have to be relative

here are the ones that need fixing:
buildbuddy/app/components/scrollbar/scrollbar
buildbuddy/app/util/animated_value
buildbuddy/app/util/animation_loop
buildbuddy/app/util/dom
buildbuddy/app/util/math

Reply:
Ah OK. Hopefully there is a way to tweak the tsconfig to get around this but I'll worry about that some other time